apps/web/components/booking/CancelBooking.tsx

```diff
@@ -71,7 +136,32 @@ export default function CancelBooking(props: Props) {
       )}
       {!error && (
         <div className="mt-5 sm:mt-6">
+          {props.isHost && teamId && (
+            <>
+              <InternalNotePresetsSelect
+                internalNotePresets={props.internalNotePresets}
+                onPresetSelect={(option) => {
+                  if (!option) return;
+
+                  if (option.value === "other") {
+                    setInternalNote({ id: -1, name: option.label });
```

Comment:
nit: could we extract -1 as a const variable and give it a name, like OTHER_FIELD or something like that?